lib/search_solr_tools/config/environments.yaml

```diff
@@ -9,17 +9,26 @@
   # should be. GLA01.018 will show up if we use DCS API v2.
   :nsidc_oai_identifiers_url: oai?verb=ListIdentifiers&metadataPrefix=dif&retired=false
 
+  # Log details.  Can be overridden by environment-specific values
+  :log_file: /var/log/search-solr-tools.log
+  :log_file_level: warn
+  :log_stdout_level: info
+
 :local:
   :host: localhost
   :nsidc_dataset_metadata_url: http://integration.nsidc.org/api/dataset/metadata/
+  :log_file: ./logs/search-solr-tools.log
 
-:dev:
+:dev: &dev
   ## For the below, you'll need to instantiate your own search-solr instance, and point host to that.
   :host: dev.search-solr.USERNAME.dev.int.nsidc.org
   ## For the metadata content, either set up your own instance of dataset-catalog-services
   ## or change the URL below to point to integration
   :nsidc_dataset_metadata_url: http://dev.dcs.USERNAME.dev.int.nsidc.org:1580/api/dataset/metadata/
 
+:development:
+  <<: *dev
```

Comment:
Did you add `development` for consistency with anything in particular?

Reply:
I could have just removed the :dev: section, but decided it didn't hurt to have both.  But in `environment.rb` it uses the default `:development` (not :dev), and I had run into some issues when trying to use the `:development` environment because of this.

I figured, having both here just means that both "dev" and "development" environment settings will behave the same way